client/src/Video.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import { AiFillLike, AiFillDislike, AiFillDelete } from "react-icons/ai";
+
+function Video({ info, handleDelete }) {
+  const [currentRating, setCurrentRating] = useState(info.rating);
+
+  function handleDeleteBtn() {
+    handleDelete(info.id);
+  }
+
+  function handleLike() {
+    if (info.id) {
+      setCurrentRating(currentRating + 1);
+      updateRating(info.id, currentRating + 1);
+    }
+  }
+
+  function handleDislike() {
+    if (info.id && currentRating > 0) {
+      setCurrentRating(currentRating - 1);
+      updateRating(info.id, currentRating - 1);
+    }
+  }
```

Comment:
I like the way you;'ve split this logic up 🎉

Reply:
Thanks a lot